pack/mut_gen.json

```diff
@@ -1060,13 +1082,15 @@
 		"resource_wild": 1,
 		"set_code": "brawler",
 		"set_position": 5,
+		"text": "<b>Hero Resource</b>: Generate [wild][wild] resources for an [[Attack]] or [[Defense]] event. Gain a tough status card. Remove this card from the game and the campaign pool.",
 		"traits": "Skill.",
 		"type_code": "upgrade"
 	},
 	{
 		"code": "32181",
 		"cost": 0,
 		"faction_code": "campaign",
+		"flavor": "\"Tally-ho!\"  — Nightcrawler",
```

Comment:
I don't know if it matters with rendering since I haven't tested this but I think we only need a single space between the quote and em dash in the flavor text. Looks like most of these have 2 instead of 1.